src/vm/decodemd.cpp

```diff
@@ -73,7 +73,7 @@ const BYTE decoded_10[2] = {10, END_DECODED };
 #define DOING_BITS (MAX_HEADER+1)
 #define DECODING_BITS(partial, got, header) (DOING_BITS+(partial<<8)+(got<<16)+(header<<24))
 #define DECODING_ERROR ((unsigned) -1)
-#define MASK(len) (~((~0)<<len))
+#define MASK(len) (~((~((unsigned int)0))<<len))
```

Comment:
To make it the same as the already checked in change for similar thing, could you please replace the cast by adding "u" suffix to the 0?